io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "vimix",
+    "rename-desktop-file": "vimix.desktop",
+    "rename-icon": "vimix",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--device=shm",
+        "--device=all",
+        "--filesystem=host",
+        "--filesystem=/tmp",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=FREI0R_PATH=/app/lib/frei0r-1"
+    ],
+    "cleanup": [
+        "/include",
+        "/share/gdb",
+        "/share/info",
+        "/share/man",
+        "/share/doc",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "glfw",
+            "buildsystem": "cmake",
```

Comment:
```suggestion
            "buildsystem": "cmake-ninja",
```

Reply:
GLFW is the base library for the windowing  and I/O system integration.